day_7/high_level.md

```diff
@@ -0,0 +1,37 @@
+## Ceasar Cipher
```

Comment:
Very nice work on both the high level steps and implementation. Great job thinking in Class structure for CeasarCipher.